src/Illuminate/Support/Str.php

```diff
@@ -779,4 +779,85 @@ public static function createUuidsNormally()
     {
         static::$uuidFactory = null;
     }
+
+    /**
+     * Clean a string with easy to remember options
+     *
+     * @param $target
```

Comment:
invalid syntax

Reply:
Ok - I think this one is now sorted.  Thanks Graham

Reply:
You must include a type. That's why the syntax is invalid.

Reply:
Ok - sorry - I got this now sorry.